alcs-api/src/main.ts

```diff
@@ -78,6 +78,9 @@ async function bootstrap() {
     new FastifyAdapter({
       logger: { level: config.get<string>('LOG_LEVEL') },
     }),
+    {
+      logger: [config.get<LogLevel>('LOG_LEVEL')],
+    },
```

Comment:
hmm., 2 loggers?

Reply:
Yes, the first option only seems to set the Fastify logging level, but does not set the NestJS logger level, Not entirely sure why it's separate?

Reply:
Even worse, you see in my MR that even setting the logging level here doesn't seem to impact keycloak even when it has useNestJsLogger:true, so I had to manually set its log level also?